lib/db/models.py

```diff
@@ -28,6 +28,11 @@ class Comment(BaseModel):
 
 class Issue(BaseModel):
     issue_id = pw.IntegerField(primary_key=True)
+    number = pw.IntegerField()
```

Comment:
The PR number as oppose to PR id? The thing is when github returns a PR, it returns a value called "id" which is not the PR number we see on github, and it returns a "number" which is the number we see on github. TBH, I am really confused about which to use where. I think we want to use issue/PR numbers, rather than ids, but the db models appear to use the ids. Any help/auditing on this point would be appreciated.